packages/api/src/utils/zod.ts

```diff
@@ -44,12 +43,15 @@ export const numberFormatSchema = z.object({
 export const timeChartSeriesSchema = z.object({
   table: z.optional(sourceTableSchema),
   type: z.literal('time'),
-  aggFn: aggFnSchema,
+  aggFn: AggregateFunctionSchema,
+  level: z.number().optional(),
   field: z.union([z.string(), z.undefined()]),
   where: z.string(),
+  whereLanguage: z.optional(z.enum(['sql', 'lucene'])),
   groupBy: z.array(z.string()).max(10),
   numberFormat: numberFormatSchema.optional(),
   metricDataType: z.optional(z.nativeEnum(MetricsDataType)),
+  metricName: z.optional(z.string()),
```

Comment:
Does this do something different that `z.string().optional()`?

Reply:
same difference. I'll make it consistent

Reply:
Addressed